src/app/information/information.component.html

```diff
@@ -1,59 +1,74 @@
 <app-project-page>
-  <h1 class="fc-display">
-    Episode Information
-  </h1>
+  <h1 class="fc-display">Episode Information</h1>
 
-  <a routerLink="/" class="fc-link">{{ 'PAGES.DETAIL.BACK_TO_HOME' | translate }}</a>
+  <a routerLink="/home" class="fc-link">{{ 'PAGES.DETAIL.BACK_TO_HOME' | translate }}</a>
 
-  <div class="p-field">
-    <label for="show-name" class="fc-body">Show Name</label>
-    <input id="show-name" class="p-inputtext-lg" aria-describedby="show-name-help" pInputText
-      [value]="(episodeInfo$ | async).showName" (input)="setShowName($event)" />
-    <small id="show-name-help" class="fc-description">The name of the show.</small>
-  </div>
+  <div *ngIf="episodeInfo">
 
-  <div class="p-field">
-    <label for="title" class="fc-body">Title</label>
-    <input id="title" class="p-inputtext-lg" aria-describedby="title-help" pInputText
-      [value]="(episodeInfo$ | async).title" (input)="setTitle($event)" />
-    <small id="title-help" class="fc-description">The title of the episode.</small>
-  </div>
+    <div class="p-field">
+      <label for="show-name" class="fc-body">Show Name</label>
+      <input id="show-name" class="p-inputtext-lg" aria-describedby="show-name-help" pInputText
+        [(ngModel)]="episodeInfo.showName"
+        (blur)="updateInfo()"
```

Comment:
Instead of saving on every keystroke, the information will be persisted to the UI Model on blur

Reply:
If we save on each blur, are we tracking whether or not the value actually changed?

Reply:
The data is "saved" in memory on every keystroke. And consequentially persisted to Store on blur. We can track if a change is not persisted by creating and maintaining a flag that tracks the "Dirty" state of the form - and perhaps ask the "You have unchanged data, continue?" kind of dialog.

But since this is the Happy Path, I wouldn't expect this covered here